Create setupToolbox

vijayiyer05 opened the pull request:
Simple generic utility that can be part of one "recipe" to allow live ready-to-run live script examples just opened to via Open in MATLAB Online

vijayiyer05 committed:
Create setupToolbox

Simple generic utility that can be part of one "recipe" to allow live ready-to-run live script examples just opened to via Open in MATLAB Online

vijayiyer05 committed:
Rename setupToolbox to setupToolbox.m

tiborauer commented on setupToolbox.m:
```diff
@@ -0,0 +1,3 @@
+% One-line script which adds /toolbox folder contents (including subfolders) to path for current MATLAB session
+% Key use case: enables specific live scripts in /examples folder opened to via Open in MATLAB Online badge to utilize library code (i.e., in /toolbox folder), for simple toolboxes (no dependencies) not currently using package management. 
+addpath(genpath("../../toolbox"));
```
Considering that the path to the toolbox root folder is hardcoded, the `setupToolbox` script MUST have a pre-specified location within the code. You may want to describe it here and/or in the README.

vijayiyer05 replied:
Good point! I've tried to clarify this, how does it seem?

vijayiyer05 committed:
Update setupToolbox.m

Added information about "how to use" including where the function should be located & called

vijayiyer05 commented:
Thank you @tiborauer for the review. I believe you have maintainer privileges. If you see the green Merge button, can you proceed? If you don't, please let me know.

tiborauer commented:
Sure. Conventionally, the submitter should/could merge it after a successful review, also a celebratory step. :)
 I am also happy to do it.
